proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -298,16 +305,14 @@ public void done(ProxyPingEvent pingResult, Throwable error)
         {
             pingBack.done( getPingInfo( motd, protocol ), null );
         }
-
-        thisState = State.PING;
     }
 
     @Override
     public void handle(PingPacket ping) throws Exception
     {
         Preconditions.checkState( thisState == State.PING, "Not expecting PING" );
         unsafe.sendPacket( ping );
-        disconnect( "" );
```

Comment:
Why?

Reply:
i thought it is unnecessary to call disconnect because we never need to check if we can send a kick packet. da wir ja im status protocol sind

Reply:
@Outfluencer what abt "because we are in the status protocol"

Reply:
@Janmm14 lol i got a brain disfunction  or something writing this